cmd/ncdu/ncdu.go

```diff
@@ -1,7 +1,7 @@
 // Package ncdu implements a text based user interface for exploring a remote
 
-//go:build !plan9 && !solaris && !js
-// +build !plan9,!solaris,!js
+//go:build !plan9 && !js
```

Comment:
Do any docs need to be updated?

Reply:
We don't usually state which subcommands work on which OSes, the OS just won't have that subcommand.

However making it run on solaris is a nice bonus :-)